controllers/auth.js

```diff
@@ -388,3 +476,34 @@ exports.resetStudentPassword = asyncHandler(async (req, res, next) => {
 
   res.status(200).json({ success: true, data: student });
 });
+
+// Update student details
+// PUT /api/v1/auth/updatedStudentdetails
+// Private
+exports.updateStudentDetails = asyncHandler(async (req, res, next) => {
+  const fieldsToUpdate = {
+    name: req.body.name,
+    email: req.body.email
+  }
+
+  const student = await Student.findByIdAndUpdate(req.student.id, fieldsToUpdate, {
+    new: true,
+    runValidators: true
+  });
+
+  res.status(200).json({
+    success: true,
+    data: student
+  });
+});
+// Get current logged in user
+//@ riute POST/api/v1/auth/me
+// @access Private
+exports.getMe = asyncHandler(async (req, res, next) => {
```

Comment:
Why do we have many  `getMe` functions? This will clash with other `getMe` functions too. Please rename your function